### PR TITLE
[bitnami/aspnet] fixed ingress configuration if using extraHosts

### PR DESCRIPTION
**Pull request by adaumann**
If having extraHosts in ASPNET ingress, the helm chart template fails with 'nil pointer evaluating interface {}' on extraPaths. Furthermore the ingress uses old names for the service link. Therefore the ingress.yaml in asp.net need to be adjusted. 

Example:
```
[...]
ingress:
  enabled: true
  selfSigned: false
  ingressClassName: nginx
  hostname: apigateway.hcv-cluster-local.net
  servicePort: http
  tls: true
  selfsigned: false
  annotations:
    nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
    #certmanager.k8s.io/issuer: "letsencrypt-prod"
    #certmanager.k8s.io/acme-challenge-type: http01    
  pathType: ImplementationSpecific
  extraHosts:
    - name: apigateway.hcv-cluster-local.net
      path: /api
  extraTls:
    - hosts:
        - apigateway.hcv-cluster-local.net
      secretName: apigateway.local-tls
  secrets: 
    - name: apigateway.local-tls
      certificate: |-
        ---- BEGIN CERT...
      key: |-
        ---- BEGIN RSA...
```
### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by adaumann — COMMENTED** [review]
See issue #10032

File: bitnami/aspnet-core/templates/ingress.yaml [hunk 1]
```diff
@@ -28,7 +27,7 @@ spec:
       http:
         paths:
           {{- if .Values.ingress.extraPaths }}
-          {{- include "common.tplvalues.render" (dict "value" .Values.ingress.extraPaths  "context" $) | nindent 10 }}
+          {{- toYaml .Values.ingress.extraPaths | nindent 10 }}
```
Comment (javsalgar) [inline bitnami/aspnet-core/templates/ingress.yaml:30]: Could you explain why we need to switch from `render` to `toYaml'?

Comment (adaumann) [reply bitnami/aspnet-core/templates/ingress.yaml:30]: With the previous settings I got an helm template error, therefore I orientated on bitnami keycloak chart and taken over the same configuration. Any suggestion?

Comment (javsalgar) [reply bitnami/aspnet-core/templates/ingress.yaml:30]: Which is the error that appeared?

Comment (adaumann) [reply bitnami/aspnet-core/templates/ingress.yaml:30]: 'nil pointer evaluating interface {}' on extraPaths.

**Review by danielgauci — APPROVED** [review]

File: bitnami/aspnet-core/templates/ingress.yaml (hunk 1 above)
Comment (adaumann) [reply bitnami/aspnet-core/templates/ingress.yaml:30]: This is basically the example from above, just need the ingress configuration with extraHosts

**Comment by adaumann** [comment]
fixed version number

**Comment by adaumann** [comment]
adopted version number

**Pull request merged**